mmedit/engine/hooks/visualization_hook.py

```diff
@@ -88,7 +88,7 @@ class GenVisualizationHook(Hook):
 
     - Required key words:
         - 'type': Value must be string. Denotes what kind of sampler is used to
-            generate image. Refers to `:meth:~mmgen.core.sampler.get_sampler`.
+            generate image. Refers to `:meth:~mmedit.core.sampler.get_sampler`.
```

Comment:
```suggestion
            generate image. Refers to :meth:`~mmedit.utils.get_sampler`.
```

latest dev-1.x has resolved it